admin_ui/src/components/CSVButton.vue

```diff
@@ -59,21 +54,27 @@ export default {
                     )
                     exportedRows.push(...response.data.rows)
                 }
-                return exportedRows
+                let csv: string = "data:text/csv;charset=utf-8,"
+                csv += [
+                    Object.keys(exportedRows[0]).join(";"),
+                    ...exportedRows.map((item) =>
+                        Object.values(item).join(";")
+                    ),
+                ]
+                    .join("\n")
+                    .replace(/(^\[)|(\]$)/gm, "")
+                const data: string = encodeURI(csv)
+                const link: HTMLAnchorElement = document.createElement("a")
+                link.setAttribute("href", data)
+                link.setAttribute("download", `${this.tableName}.csv`)
+                link.click()
+                alert("Your data is ready.")
```

Comment:
It looks like you were able to replicate the download to CSV without using the library. Is this correct?

Reply:
Yes. It does the same thing without an external library because `vue-json-excel` doesn't have support for Vue 3 and based on repo activities there probably won't be any support soon.